pages/users/getting-started/introduction.md

```diff
@@ -1 +1,41 @@
-# Introduction
\ No newline at end of file
+# Introduction
```

Comment:
This page is too generic and repetitive. It says same things as introduction section. Instead, add an introduction for the users that want to access to XRPL EVM. Write something like, to access all the fuctionallities of the XRPL EVM you will need an account and XRP. To get that you will need a wallet and use the bridge, in this section you will learn how to do it.